register.html

```diff
@@ -50,18 +50,18 @@ <h2 class="reg_head"> Register here. </h2>
             <input type="email" class="regin_form" name="email_id" placeholder="Email id" required="email">
             <input type="password" class="regin_form" name="password" placeholder="Password" required="password"><br>
             <input type="radio" class="regin_form radio_btn" name="gender" value="Male" checked="" style="margin-left: 25%;">Male
-            <input type="radio" class="regin_form radio_btn" name="gender" value="Female">Female<br>
+            <input type="radio" class="regin_form radio_btn" name="gender" value="Female" id="female">Female<br>
             <select class="regin_form" required="">
-            <option value="" disabled="" selected="" hidden="">Proficiency Level</option>
-            <option value="Novice">Novice</option>
-            <option value="Intermediate">Intermediate</option>
-            <option value="Advanced">Advanced</option>
-            <option value="Expert">Expert</option>
-          </select>
+            	<option value="" disabled="" selected="" hidden="">Proficiency Level</option>
+            	<option value="Novice">Novice</option>
+            	<option value="Intermediate">Intermediate</option>
+            	<option value="Advanced">Advanced</option>
+            	<option value="Expert">Expert</option>
+            </select>
             <input type="submit" id="reg_btn" value="Create an Account">
         </form>
     </div>
-    <footer>
+    <footer style="background-color: #ececec">
         <div class="container">
             <div class="footer_child">
                 <img src="https://upload.wikimedia.org/wikipedia/en/thumb/8/8c/Facebook_Home_logo_old.svg/1024px-Facebook_Home_logo_old.svg.png" width="23" height="23" class="footer_img" alt="Join our Facebook page! (267k)" onclick="location.href='http://www.facebook.com'" ">
```

Comment:
It seems a lot of information inside <img> tag. It can be simplified. Use meaningful text for 'alt'.